openupgrade_scripts/scripts/mail/16.0.1.10/pre-migration.py

```diff
@@ -124,7 +124,7 @@ def _update_mail_channel_name(env):
         WITH sub AS(
             SELECT res_id, value FROM _ir_translation
             WHERE name = 'mail.channel,name'
-            AND value IS NOT NULL
+            AND (value IS NOT NULL OR value != '')
```

Comment:
@duong77476 `và` chứ sao lại `hoặc`
```suggestion
            AND value IS NOT NULL AND value != ''
```

Reply:
À vâng anh e nhầm